internal/v2/repository/project/member.go

```diff
@@ -3,74 +3,102 @@ package project
 import (
 	"context"
 	"github.com/caos/zitadel/internal/eventstore/v2"
+	"github.com/caos/zitadel/internal/eventstore/v2/repository"
 	"github.com/caos/zitadel/internal/v2/repository/member"
 )
 
 var (
-	MemberAddedEventType   = projectEventTypePrefix + member.AddedEventType
-	MemberChangedEventType = projectEventTypePrefix + member.ChangedEventType
-	MemberRemovedEventType = projectEventTypePrefix + member.RemovedEventType
+	ProjectMemberAddedType   = projectEventTypePrefix + member.AddedEventType
+	ProjectMemberChangedType = projectEventTypePrefix + member.ChangedEventType
+	ProjectMemberRemovedType = projectEventTypePrefix + member.RemovedEventType
```

Comment:
i now in other packages it is like this, but prefixing with package name is bad practice

Reply:
yes i think this was, because we moved the files a lot in the beginning, we should check this for all files before we are finished